onnxruntime/core/providers/webnn/builders/impl/gemm_op_builder.cc

```diff
@@ -30,6 +30,72 @@ class GemmOpBuilder : public BaseOpBuilder {
                                const logging::Logger& logger) const override;
 };
 
+// Helper function
+common::Status ProcessZeroPointAndScale(
+    const std::vector<int64_t>& input_shape,
+    const size_t zero_point_index,
+    emscripten::val& zero_point,
+    emscripten::val& scale,
+    ModelBuilder& model_builder,
+    const Node& node,
+    const int32_t zero_point_type,
+    const logging::Logger& logger) {
```

Comment:
```suggestion
    ModelBuilder& model_builder,
    const Node& node,
    int32_t zero_point_type,
    const logging::Logger& logger,
    emscripten::val& zero_point,
    emscripten::val& scale) {
```
Output parameters really should go last.